helpdesk_mgmt/README.rst

```diff
@@ -172,6 +172,7 @@ Contributors
   * Carlos Martínez
   * Catalin Airimitoaie
   * Álvaro López
+  * Nacho Serra
```

Comment:
Don't do this here, but on readme/CONTRIBUTORS.rst file

Reply:
Changed, sorry